bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties

```diff
@@ -0,0 +1,9 @@
+bin.includes = META-INF/,\
+               OSGI-INF/,\
+               .,\
+               about.html,\
+               lib/libdbus-java-2.7.jar,\
+               lib/unix-0.5.jar

```

Comment:
> On 16 Dec 2015, at 11:03, Kai Kreuzer <notifications@github.com> wrote:
> 
> Do you possibly have a link to the official sources?

No - but I’ll see what I can find. I think the problem is that the author is not too active any more (from what I read on the dbus list) so there are many sources, and an ‘official’ one might be difficult to find...